tests/test_user.py

```diff
@@ -1,26 +1,73 @@
 # This module tests to see APIs that relates to users works correctly or not
+# NOTE: This test should not be used in production server
+# (Due to generating random user in DB)
+# TODO: Make test to be usable in production server
 import json
+import random
+import string
 
 # Load config file
 with open('config.json', mode='r') as config_file:
     CONFIG = json.load(config_file)
-
 HEADERS = {
     'Content-Type': 'application/json'
 }
-
 TOKEN = ""
 
 
+def generate_random_string(length):
+    """
+    For generating random username
+    """
+    letters = string.ascii_lowercase
```

Comment:
Remove this variable and use the `string.ascii_lowercase` directly in below line.